src/pyskindose/geom_calc.py

```diff
@@ -1,17 +1,62 @@
 import logging
 from typing import List, Any
+import pyskindose.constants as c
```

Comment:
Move to below imports of third party packages

Reply:
resolved